Vagrantfile

```diff
@@ -24,94 +25,20 @@ Vagrant.configure("2") do |config|
     config.vm.define node[:name] do |cplane|
       cplane.vm.box = node[:box]
       cplane.vm.network node[:network], ip: node[:ip]
-      cplane.vm.network "forwarded_port", guest: 443, host: 8443 # Port Forward for k8s dashboard
```

Comment:
This was under the assumption that it could be used for the dashboard, but removed as it is not being used